docs/pages/versions/unversioned/sdk/network.md

```diff
@@ -43,8 +43,9 @@ A `Promise` that resolves to an object with the following fields:
 
 - **type (_NetworkStateType_)** -- a [`NetworkStateType`](#networknetworkstatetype) enum value that represents the current network connection type.
 
-- **isConnected (_boolean_)** -- if there is an active network connection. Note that this does not mean that internet is reachable.
-- **isInternetReachable (_boolean_)** -- if the internet is reachable with the currently active network connection.
+- **isConnected (_boolean_)** -- if there is an active network connection. Note that this does not mean that internet is reachable. On Android, this returns `false` if the `type` is either `Network.NetworkStateType.NONE` or `Network.NetworkStateType.UNKNOWN`, `true` otherwise.
```

Comment:
> On Android

This commit implements this logic for iOS. This documentation probably needs work to be accurate and complete.